### PR TITLE
Formatting, minor wording changes

### PR DESCRIPTION
**Pull request by vszal**

### CODE REVIEW

File: containerd-http-proxy/README.md [hunk 1]
```diff
@@ -1,13 +1,16 @@
 # Configure Containerd HTTP/S proxy
 
-This guide outlines the steps to configure the HTTP/S proxy for Containerd within a Kubernetes environment, specifically tailored for GKE Autopilot clusters.
+This guide outlines the steps to configure a HTTP/S proxy for Containerd on GKE nodes, including Autopilot mode clusters. Typical use cases include access of external image repositories for container pulls.
 
 ## Instruction
```
Comment (vinayakankugoyal) [inline containerd-http-proxy/README.md]: nit: Instructions

File: containerd-http-proxy/README.md [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 
-1. Create a ConfigMap named containerd-proxy-configmap that includes the values for HTTP_PROXY, HTTPS_PROXY, and NO_PROXY (optional). These values are used as environment variables to configure the proxy settings for the Containerd service. A sample ConfigMap configuration is provided in sample_configmap.yaml. Please modify this sample with your proxy settings before applying it to your cluster.
-2. Deploy the daemonset in configure_http_proxy.yaml. As it has been specifically allowlisted for GKE Autopilot, the manifest shouldn’t be changed to make sure it can be deployed successfully in GKE Autopilot clusters.
+1. Create a ConfigMap named `containerd-proxy-configmap` that includes the values for `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` (optional). These values are used as environment variables to configure the proxy settings for the Containerd service. A sample ConfigMap configuration is provided in `sample_configmap.yaml`. Please modify this sample with your proxy settings before applying it to your cluster.
+
+`kubectl apply -f sample_configmap.yaml`
```
Comment (vinayakankugoyal) [inline containerd-http-proxy/README.md:9]:
```suggestion
```
kubectl apply -f sample_configmap.yaml
```
```

Comment (vinayakankugoyal) [inline containerd-http-proxy/README.md]: use code block instead. replace with "`" with "```"

File: containerd-http-proxy/README.md [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
+
+2. Deploy the daemonset in `configure_http_proxy.yaml`. As it has been specifically allowlisted for GKE Autopilot, this **manifest in this repo cannot be changed if you are deploying to GKE Autopilot mode clusters**.
+
+`kubectl apply -f configure_http_proxy.yaml`
```
Comment (vinayakankugoyal) [inline containerd-http-proxy/README.md]: use code block instead. replace with "`" with "```"

**Review by vszal — COMMENTED** [review]
Fixes per [vinayakankugoyal](https://github.com/vinayakankugoyal) suggestions

File: containerd-http-proxy/README.md (hunk 3 above)
Comment (vszal) [reply containerd-http-proxy/README.md]: updated

File: containerd-http-proxy/README.md (hunk 2 above)
Comment (vszal) [reply containerd-http-proxy/README.md]: updated

File: containerd-http-proxy/README.md (hunk 1 above)
Comment (vszal) [reply containerd-http-proxy/README.md]: fixed

File: containerd-http-proxy/README.md [hunk 4]
```diff
@@ -1,13 +1,16 @@
...
 
-## Instruction
+## Instructions
 
-1. Create a ConfigMap named containerd-proxy-configmap that includes the values for HTTP_PROXY, HTTPS_PROXY, and NO_PROXY (optional). These values are used as environment variables to configure the proxy settings for the Containerd service. A sample ConfigMap configuration is provided in sample_configmap.yaml. Please modify this sample with your proxy settings before applying it to your cluster.
-2. Deploy the daemonset in configure_http_proxy.yaml. As it has been specifically allowlisted for GKE Autopilot, the manifest shouldn’t be changed to make sure it can be deployed successfully in GKE Autopilot clusters.
+1. Create a ConfigMap named `containerd-proxy-configmap` that includes the values for `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` (optional). These values are used as environment variables to configure the proxy settings for the Containerd service. A sample ConfigMap configuration is provided in `sample_configmap.yaml`. Please modify this sample with your proxy settings before applying it to your cluster.
+
+```kubectl apply -f sample_configmap.yaml```
```
Comment (vinayakankugoyal) [inline containerd-http-proxy/README.md:9]: this needs to be formatter like

```
kubectl apply -f sample_configmap.yaml
```

Comment (vinayakankugoyal) [inline containerd-http-proxy/README.md]:
```suggestion
```
kubectl apply -f sample_configmap.yaml
```
```

File: containerd-http-proxy/README.md [hunk 5]
```diff
@@ -1,13 +1,16 @@
...
+
+2. Deploy the daemonset in `configure_http_proxy.yaml`. As it has been specifically allowlisted for GKE Autopilot, this **manifest in this repo cannot be changed if you are deploying to GKE Autopilot mode clusters**.
+
+```kubectl apply -f configure_http_proxy.yaml```
```
Comment (vinayakankugoyal) [inline containerd-http-proxy/README.md]:
```suggestion
```
kubectl apply -f configure_http_proxy.yaml
```
```

File: containerd-http-proxy/README.md (hunk 4 above)
Comment (vinayakankugoyal) [reply containerd-http-proxy/README.md]: That comment didn't render correctly but add a new line after the first "````" and the second "```"

**Review by vinayakankugoyal — APPROVED** [review]

**Review by vinayakankugoyal — APPROVED** [review]

**Comment by vszal** [comment]
abandoning ship

**Pull request closed without merging**